reporting_app/__init__.py

```diff
@@ -32,22 +32,41 @@ def pipeline_report(pipeline_type, view_type):
 
     if view_type == 'all':
         query = rest_query(endpoint)
+        run_data = query_api('aggregate/all_runs')
+        sample_data = query_api('aggregate/samples')
+
     elif view_type in statuses:
         query = rest_query(endpoint, match={'$or': [{'proc_status': s} for s in statuses[view_type]]})
+        args = ({'$or': [{'proc_status': s} for s in statuses[view_type]]})
+        run_data = query_api('aggregate/all_runs', match=args)
+        sample_data = query_api('aggregate/samples', match=args)
```

Comment:
This results in 2 API queries, but only 1 is ever used. We should decide whether to render a sample or run view, then to API queries and render templates accordingly.
(API queries should also go through `egcg_core` once the `apache` branch is merged)